.dockerignore

```diff
@@ -0,0 +1,25 @@
+**/.classpath
+**/.dockerignore
+**/.env
+**/.git
+**/.gitignore
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
+**/*.*proj.user
+**/*.dbmdl

```

Comment:
why this complex dockerignore?

Reply:
Auto-generated by the IDE (a vscode extension)
I don't mind removing this stuff if you prefer